core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/DataColumnType.kt

```diff
@@ -44,13 +49,26 @@ public inline fun <reified T> AnyCol.isType(): Boolean = type() == typeOf<T>()
 
 public fun AnyCol.isNumber(): Boolean = isSubtypeOf<Number?>()
 
+public fun AnyCol.isBigNumber(): Boolean = isSubtypeOf<BigInteger?>() || isSubtypeOf<BigDecimal?>()
+
 public fun AnyCol.isList(): Boolean = typeClass == List::class
 
+/** @include [isInterComparable] */
+@Deprecated(
+    message = IS_COMPARABLE,
+    replaceWith = ReplaceWith(IS_COMPARABLE_REPLACE, IS_INTER_COMPARABLE_IMPORT),
+    level = DeprecationLevel.WARNING,
+)
+public fun AnyCol.isComparable(): Boolean = isInterComparable()
+
 /**
- * Returns `true` if [this] column is comparable, i.e. its type is a subtype of [Comparable] and its
- * type argument is not [Nothing].
+ * Returns `true` if [this] column is inter-comparable, i.e.
+ * its elements can be compared with each other.
+ *
+ * Technically, this means the elements' common type is a subtype of [Comparable] with
+ * the type argument not being [Nothing].
  */
-public fun AnyCol.isComparable(): Boolean =
+public fun AnyCol.isInterComparable(): Boolean =
```

Comment:
isInterComparable is a new / not widespread term, for me it raises some questions like what's the difference (if i were to see it for the first time)
For users isComparable can be described as "filter columns for which max(), min(), median() and other (we actually only have 3) order operations make sense"
You wouldn't expect to find max() value for mixed column of String and Double values, so it should make things clearer

Reply:
Well, that's actually the reason I changed it, because for me `isComparable` can be interpreted in two ways:
- "is comparable": so can call max(), min(), etc., not true for mixed number columns
- "is `Comparable`"; implements the interface, which would be true for mixed number columns

you need KDocs to explain what it actually means. inter-comparable means "comparable inside" (or something, I came up with it too), which narrows down the meaning to the first. Of course, other name suggestions are very welcome if you can think of any :)

Reply:
With suggestion of ChatGPT I'll rename it to `.valuesAreComparable()`. Would that be okay?